docs/getting-started.md

```diff
@@ -74,7 +74,75 @@ To create a new headless WordPress app using SnapWP, follow these steps:
 
 ### Manual Installation
 
-@todo
+1. Install npm packages in your Next App
+
+```bash
+npm i @snapwp/blocks @snapwp/core @snapwp/next @snapwp/query
+```
+
+2. Copy the .env contents from `Dashboard > WPGraphQL > Settings > SnapWP Helper` from the WP server and create an `.env` file in the project root.
```

Comment:
Reverse the order: first you create an env file, and then you copy-paste the contents into it.